secator/cli_helper.py

```
@@ -69,6 +71,52 @@
+			if finding_type == otype.get_name():
+				klass = otype
+				break
+
+		if klass:
+			try:
+				item = klass.load(obj)
+				# Preserve the original _uuid if it exists, otherwise it will be generated
+				if '_uuid' in obj:
+					item._uuid = obj['_uuid']
+				results.append(item)
+			except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)